docs/administration/netbox-shell.md

```diff
@@ -105,6 +105,8 @@ This approach can span multiple levels of relations. For example, the following
 >>> IPAddress.objects.filter(interface__device__site__region__slug="north-america")
 ```
 
+The `get()` method can be used in place of the `filter()` method if we expect this query to return only one object. This method will yield the actual object resulting from the query, instead of a QuerySet. If the query returns more than one result, it will raise an exception.
```

Comment:
This is redundant to information provided in the [Django documentation](https://docs.djangoproject.com/en/5.0/ref/models/querysets/). Please reference it here instead of duplicating text that we'll need to then maintain.